docs/source/conf.py

```diff
@@ -108,6 +110,16 @@
     "torch.utils.data.datapipes.map.combinatorics.T_co": "T_co",
     "torchdata.datapipes.iter.util.cycler.T_co": "T_co",
     "torchdata.datapipes.iter.util.paragraphaggregator.T_co": "T_co",
+    "torchdata.datapipes.map.util.cacheholder.T_co": "T_co",
+    "Sequence[torchdata.datapipes.map.util.unzipper.T]": "Sequence[T]",
+    "torchdata.datapipes.iter.util.samplemultiplexer.T_co": "T_co",
+    "torchdata.datapipes.iter.util.indexadder.K": "K",
+    "torchdata.datapipes.iter.util.unzipper.T": "T",
+    "torch.utils.data.datapipes.iter.grouping.T_co": "T_co",
+    "torchdata.datapipes.iter.util.dataframemaker.T_co": "T_co",
+    "torchdata.datapipes.iter.util.cacheholder.T_co": "T_co",
+    "torchdata.datapipes.iter.util.header.T_co": "T_co",
+    "<class 'torch.utils.data.datapipes.datapipe.DataChunk'>": "torch.utils.data.datapipes.datapipe.DataChunk",
```

Comment:
Could we remove `DataChunk` from the doc? This is not supposed to be exposed to users IMHO.